hack/load-tests/log-fluentbit-test-setup.yaml

```diff
@@ -76,6 +76,18 @@ spec:
       labels:
         app: logs-load-generator
     spec:
+      affinity:
```

Comment:
That will improve the situation, I agree, still it can be unequal distribution when having 10 pods on 6 nodes. Shouldn't we simply switch to a daemonset and keep the amount of pods always equal to the nodes and then increase/decrease load by config options of the pod?

Reply:
most load tests require more load generator pods, we have a 2 node setup for all test scenarios I think deployment will evenly distribute pods on two nodes